core/trino-main/src/test/java/io/trino/sql/planner/TestLiteralEncoder.java

```diff
@@ -239,7 +238,6 @@ public void testEncodeTimestampWithTimeZone()
     @Test
     public void testEncodeRegex()
     {
-        assertRoundTrip(castVarcharToJoniRegexp(utf8Slice("[a-z]")), LIKE_PATTERN, (left, right) -> left.pattern().equals(right.pattern()));
```

Comment:
They are used to make sure we can encode patterns properly when converting expressions to textual form. It's because like and regex patterns get converted to specialized types under the covers, so reversing that takes a bit of work.